.circleci/config.yml

```diff
@@ -227,57 +260,75 @@ jobs:
             chmod +x ./scripts/publish_model.sh
             ./scripts/publish_model.sh ./packages/neural_network_model/
 
+
+tags_only: &tags_only
+  filters:
+    branches:
+      ignore: /.*/
+    tags:
+      only: /^.*/
+
 workflows:
   version: 2
-  test-all:
+  section_07:
     jobs:
-      - test_regression_model_py36
-      - test_regression_model_py37
-      - test_regression_model_py38
-      - test_ml_api_py36
-      - test_ml_api_py37
-      # - test_ml_api_py38 pending NN model update
-      - section_9_differential_tests
-      - train_and_upload_regression_model:
+      - section_07_test_app
+      - section_07_deploy_app_to_heroku:
           requires:
-            - test_regression_model_py36
-            - test_regression_model_py37
-            - test_regression_model_py38
-            - test_ml_api_py36
-            - test_ml_api_py37
-            - section_9_differential_tests
-          filters:
-            branches:
-              only:
-                - master
-      # - section_10_deploy_to_heroku:
-      #     requires:
-      #       - train_and_upload_regression_model
-      #     filters:
-      #       branches:
-      #         only:
-      #           - master
+            - section_07_test_app
+      # upload after git tags are created
+      - section_07_test_and_upload_regression_model:
```

Comment:
This means that the model will be published when a new tag is created via the github UI. This should reduce confusion about updating the model before/after deploying the API